### DISCUSSION TITLE
想问一下typescript重构进度如何？

### DISCUSSION THREAD

**Discussion by KID-joker** [liriliri/eruda#289] [General]
之前在issue看到说正在用typescript重构项目，想了解一下目前进度怎么样，如果不方便说，请告知。 @surunzi

**Comment by surunzi** [comment]
没有对这个项目用 ts 重构的打算，只是将部分组件抽离到另一个 [ui 组件仓库](https://github.com/liriliri/luna)，方便复用跟开发，这个仓库是用 ts 开发的。

**Reply by KID-joker** [reply]
ok，了解了，感谢回复
